app/helpers/application_helper.rb

```diff
@@ -78,4 +78,10 @@ def site_logo_url
     # There should always be a current org, but being defensive here helps simplify tests
     Organization.instance_owner&.logo_url.presence || asset_pack_path('media/images/logo.png')
   end
+
+  def present(obj, decorator, &block)
+    decorator.public_send(:new, obj, &block)
```

Comment:
I think we could call `decorator.new` directly here instead of needing `public_send`?